src/app/showcase/card-showcase/card-showcase.component.ts

```diff
@@ -23,6 +23,18 @@ export class CardShowcaseComponent {
       defaultValue: 'null',
       inputValues: ['string'],
     },
+    {
+      name: 'isHighlighted',
+      description: '(Optional) Adds additional elevation of z4 on the card.',
+      defaultValue: 'false',
+      inputValues: ['boolean'],
+    },
+    {
+      name: 'isDisabled',
```

Comment:
Consider if this property should be called `isFlat`, `onCanvas` or similar. `isDisabled` sounds a bit harsh :)
(I'll talk with @bommariusser  about it)